### ISSUE TITLE
Update outdated server libraries

### ISSUE THREAD

**Issue by mkalam-alami** [alakajam-team/alakajam#414]
> Prerequisite: TypeScript migration #411

Simply running `npm update` doesn't resolve upgrading major versions (due to semantic versioning). This would require a manual pass.

*EDIT: Remaining libs as of 04/02/2020:*

* tslint => eslint, will do a separate issue

**Not upgrading**

* Bookshelf (see #478)
* Flipclock (latest version has major changes, website seems down and repo is not active anyway. Keeping the old version is acceptable, alternately there is https://www.npmjs.com/package/flipdown)
